ui/components/ui/toggle-button/toggle-button.component.js

```diff
@@ -84,7 +105,7 @@ const ToggleButton = (props) => {
         trackStyle={value ? trackStyle : offTrackStyle}
         thumbStyle={thumbStyle}
         thumbAnimateRange={[3, 18]}
-        colors={colors}
+        colors={theme === 'light' ? colorsLight : colorsDark}
         passThroughInputProps={{
           'data-testid': dataTestId,
         }}
```

Comment:
Updating the `ToggleButton` to align with the design system Figma component

![Screenshot 2024-04-17 at 9 36 27 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/67094e1b-bad6-41ed-abbb-1f015c24b2f3)
![Screenshot 2024-04-17 at 9 36 31 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/b430887f-fc9c-4e5f-8dc8-d490676f47c3)


### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/94f0ec77-7b94-4b70-ba7c-f9968ad7efca



### After

Storybook after updates

https://github.com/MetaMask/metamask-extension/assets/8112138/4b904020-356a-40c3-9b86-fe38a586d2dd

App after changes


https://github.com/MetaMask/metamask-extension/assets/8112138/b20f68a4-9764-43b1-ac9c-ef43f5720bd6

Reply:
Moved to https://github.com/MetaMask/metamask-extension/pull/24090